src/mapper_node.cpp

```diff
@@ -106,8 +114,25 @@ void gotInput(const PM::DataPoints& input, const std::string& sensorFrame, const
 			sensorToMapBeforeUpdate = robotPoseToSet * sensorToRobot;
 			hasToSetRobotPose = false;
 		}
-
-		mapper->processInput(input, sensorToMapBeforeUpdate, std::chrono::time_point<std::chrono::steady_clock>(std::chrono::nanoseconds(timeStamp.toNSec())));
+		try
+		{
+			mapper->processInput(input, sensorToMapBeforeUpdate,
+								 std::chrono::time_point<std::chrono::steady_clock>(std::chrono::nanoseconds(timeStamp.toNSec())));
+		}
+		catch (const PM::ConvergenceError& e)
```

Comment:
A more explicit variable name (e.g., `convergenceError`) for the exception is necessary since there is another exception variable with the same name below.